client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx

```diff
@@ -143,40 +143,41 @@ export function DomainCodePair( {
 		}
 	}, [ shouldReportError, valid, domain, message, errorStatus ] );
 
-	const domainActions = ( inputValidationTextDisplayed = true ) => (
+	const domainActions = () => (
 		<>
-			{ inputValidationTextDisplayed ? <span>&nbsp;</span> : '' }
```

Comment:
Moved the spacing added by `<span>&nbsp;</span>` to CSS and got rid of `inputValidationTextDisplayed`